### PR TITLE
Rewrite code generation based on package.Load

### LINKED ISSUE #4: Rewrite code generation based on package.Load
Instead of a submodule, we should use `package.Load` to load AWS package information. Apart from getting rid of the submodule, it would make sure that code is generated against the same AWS SDK version that's required by `go.mod`.

### PR DESCRIPTION
**Pull request by sagikazarmark**
This PR provides a new code parsing mechanism that doesn't require adding the AWS SDK source as a git submodule. In theory, it should provide more correct results as it will use the same SDK version as the Go module itself.

Fixes #4

Proof that it works the same way: running the generator on master (after #8 is merged) and on this branch should give the same result (no diff in generated code).

### CODE REVIEW

File: cmd/temporal-aws-sdk-gen/internal/parser.go [hunk 1]
```diff
@@ -14,31 +19,45 @@ var blacklistedMethods = map[string]string{
...
+			packages.NeedImports |
+			packages.NeedTypes |
+			packages.NeedTypesSizes |
+			packages.NeedTypesInfo,
+	}
+
 	if serviceName == "" {
-		serviceDir := p.SdkDirectory + "/service"
-		files, err := ioutil.ReadDir(serviceDir)
+		allPkgs, err := packages.Load(config, "github.com/aws/aws-sdk-go/service/...")
```
Comment (mfateev) [inline cmd/temporal-aws-sdk-gen/internal/parser.go:37]: Let's move the repository path into a variable.
Also, need to be able to specify branch/sha when invoking the generator.

Comment (mfateev) [reply cmd/temporal-aws-sdk-gen/internal/parser.go:37]: I'm OK to add this feature in a separate PR.

**Review by mfateev — APPROVED** [review]

File: cmd/temporal-aws-sdk-gen/internal/parser.go (hunk 1 above)
Comment (sagikazarmark) [reply cmd/temporal-aws-sdk-gen/internal/parser.go:37]: I think branch/sha should be the same as the version in the module the generator runs. But let's discuss it in a different PR/issue.

Comment (mfateev) [reply cmd/temporal-aws-sdk-gen/internal/parser.go:37]: I missed that it already uses the version from the module.

**Pull request merged**